<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionnaireManagementWidget</class>
 <widget class="QWidget" name="QuestionnaireManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
}
QPushButton {
border:none;
color: black;
padding:10px;
}

QPushButton:checked {
color: blue;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="text">
         <string>问卷管理</string>
        </property>
        <property name="icon">
         <iconset resource="res/mainwindow.qrc">
          <normaloff>:/mainwindow/2.png</normaloff>:/mainwindow/2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>答卷统计</string>
        </property>
        <property name="icon">
         <iconset resource="res/mainwindow.qrc">
          <normaloff>:/mainwindow/2.png</normaloff>:/mainwindow/2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>题库管理</string>
        </property>
        <property name="icon">
         <iconset resource="res/mainwindow.qrc">
          <normaloff>:/mainwindow/2.png</normaloff>:/mainwindow/2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>打分中心</string>
        </property>
        <property name="icon">
         <iconset resource="res/mainwindow.qrc">
          <normaloff>:/mainwindow/2.png</normaloff>:/mainwindow/2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>808</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="subPage1">
      <property name="styleSheet">
       <string notr="true">QWidget#subPage1{
background-color: rgb(255, 255, 255);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>搜索：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>请输入问卷名称</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>创建时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>至</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit_2">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>618</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="add_questionnaire">
            <property name="text">
             <string>新建问卷</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1109</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="questionnaireListWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>65</width>
            <height>18</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#questionnaireListWidget{
background-color: rgb(255, 255, 255);
}</string>
          </property>
          <layout class="QVBoxLayout" name="questionnaireListLayout">
           <property name="spacing">
            <number>20</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="subPage2">
      <property name="styleSheet">
       <string notr="true">QWidget#subPage2{
background-color: rgb(255, 255, 255);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>问卷名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>请输入问卷名称</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>答卷数量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>从小到大</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>从大到小</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>294</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="answerListWidget" native="true">
         <layout class="QVBoxLayout" name="answerListLayout"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="subPage3">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>新增题目</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>975</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>按题型筛选</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>无筛选</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>打分题</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>选择题</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>主观题</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <column>
             <property name="text">
              <string>按指标筛选</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>无筛选</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>元素</string>
             </property>
             <item>
              <property name="text">
               <string>字符</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>缩写词</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>图标</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>刻度线</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>边框箭头</string>
              </property>
             </item>
            </item>
            <item>
             <property name="text">
              <string>色彩</string>
             </property>
             <item>
              <property name="text">
               <string>状态指示</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>图形和图表</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>按键和控件</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>文字和标签</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>背景和界面整体</string>
              </property>
             </item>
            </item>
            <item>
             <property name="text">
              <string>界面</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>信息</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>交互</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_bank">
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="subPage4">
      <property name="styleSheet">
       <string notr="true">QTabBar::tab {
	color: black;
	padding: 3px;
	margin-left:3px;
	background-color: transparent;
	width: 80px;
    height:40px;
}
QTabBar::tab:selected, QTabBar::tab:last:selected, QTabBar::tab:hover {
	color: blue;
	padding-left: 3px;
	padding-bottom: 2px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>问卷打分</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QWidget" name="widget_9" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>问卷名称：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="placeholderText">
                 <string>请输入问卷名称</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>搜索</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>882</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea_3">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scoringCenterListWidget">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>1205</width>
                <height>718</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="scoringCenterListLayout"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>题目打分</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="subPage5">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>基本信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>问卷名称：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>问卷描述：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" rowspan="2">
           <widget class="QTextEdit" name="questionnaire_description"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="questionnaire_name"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>添加题目</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="add_question_type2">
               <property name="text">
                <string>添加选择题</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="add_question_type1">
               <property name="text">
                <string>添加打分题</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="add_question_type3">
               <property name="text">
                <string>添加主观题</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="add_question_type4">
               <property name="text">
                <string>从题库添加</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="questionListWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>45</width>
               <height>18</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="questionListLayout">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1111</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="finish_questionnaire">
            <property name="text">
             <string>完成问卷</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="finish_edit">
            <property name="text">
             <string>编辑完成</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
